quirks/password-rules.json

```diff
@@ -23,6 +23,9 @@
     "angieslist.com": {
         "password-rules": "minlength: 6; maxlength: 15;"
     },
+    "arm.com": {
+        "password-rules": "minlength: 8; maxlength: 16; required: upper; required: lower; required: digit; required: allowed: [@#$%^&*-_+=[]{}|\\:',?/`~\"();.];"
```

Comment:
This doesn’t parse as a valid rule on `https://developer.apple.com/password-rules/`.

Maybe you want this?

```
minlength: 8; maxlength: 16; required: upper; required: lower; required: digit; allowed: [@#$%^&*-_+=[{}|\\:',?/`~\"();.]];
```

Reply:
Hm, is there a way to add [ and ] to the accepted chars? Because I've tried \\[ as well and it didn't work as far as i remember

Reply:
`]` has to be the last character in a character set.

Reply:
Yep I know. But what should we do if this char is a valid one? How can I add it to the allowed chars string? 🤔

Reply:
You make it the last character in the set, before the set closes!

```
allowed: [@#$%^&*-_+=[{}|\\:',?/`~\"();.]];
```

See how there are two `]` in a row? The first one is the one in the set. The second one is the one that closes the set.

Reply:
Aaah, got you! Thanks haha. I will add this tomorrow to the interface and to this commit 😬

Reply:
Is `required: allowed: [...]` a valid rule? I see one other example in the list for `arm.com` but not other rules specified that way. [Reading the spec](https://developer.apple.com/documentation/security/password_autofill/customizing_password_autofill_rules) it seems you should use `required`, no? Is anyone else sure about this?

Reply:
@bradcush is right. That’s not valid.

Reply:
Very soon, now that WWDC is over, we should hopefully be able to have some validation running in the CI. I’m really sorry that it’s taken a while to get up and running, but really really appreciate everyone who has been working through it.